filament/src/materials/dof/dof.mat

```diff
@@ -658,6 +659,7 @@ void postProcess(inout PostProcessInputs postProcess) {
     // the downside of doing this is that we couldn't use a different upscaler for each
     // layer, but this is a lot less costly
     postProcess.color = foreground + (1.0 - fgOpacity) * background;
+    postProcess.color = min(vec4(65504.0f), postProcess.color);
```

Comment:
Did you check that doing the `min` here is not enough? I wonder if the one on line 480 is needed if we have this one.

Reply:
Oh sorry, I didn't noticed your comment. Do you mean that I have to use `max(vec4(MEDIUMP_FLT_MIN, x)` also?  
And line 480 is necessary. If I omit it, black squares are shown inside the rings.

Reply:
yes, use `MEDIUMP_FLT_MIN`. thanks.